zdm/survey.py

```diff
@@ -1285,3 +1319,219 @@ def vet_frb_table(frb_tbl:pandas.DataFrame,
             raise ValueError(f'{field} is missing in your table!')
         elif fill:
             frb_tbl[field] = None
+
+############ We now define some width/scattering functions ############
+# These all return p(w) dlogw, and must take as arguments np.log10(widths)
+
+def quadrature_convolution(width_function, width_args, scat_function, scat_args,
```

Comment:
move out of `survey.py` ?

Reply:
I'm keeping all these here for now, simply because I am not sure where I'd put them. One possible file would be a "width_scat_functions.py" file, which keeps all this associated paraphenalia. But there's also value in reducing the number of total files - and so far, these functions are only used within survey. When one of them gets used outside survey, then it'll be time to port